harness/tests/localnet_test.go

```diff
@@ -107,7 +107,14 @@ func TestSendBidTransaction(t *testing.T) {
 	t.Log("Send request to node 0 to generate and process a Bid transaction")
 	txidBytes, err := localNet.SendBidCmd(0, 10, 10)
 	if err != nil {
-		t.Fatal(err.Error())
+		t.Fatal(err)
+	}
+
+	for i := 1; i < localNet.Size(); i++ {
```

Comment:
so we are now going to always test with sending bids to all participants by default ?

Reply:
Yes, that's the idea. Such situation is closer to what you can expect in mainnet (multiple bidders network)